openedx/core/djangoapps/content_libraries/views.py

```diff
@@ -708,9 +710,12 @@ def put(self, request, usage_key_str, file_path):
         )
         file_wrapper = request.data['content']
         if file_wrapper.size > 20 * 1024 * 1024:  # > 20 MiB
-            # In the future, we need a way to use file_wrapper.chunks() to read
-            # the file in chunks and stream that to Blockstore, but Blockstore
-            # currently lacks an API for streaming file uploads.
+            # TODO: This check was written when V2 Libraries were backed by the Blockstore micro-service.
+            #       Now that we're on Learning Core, do we still need it? Here's the original comment:
+            #         In the future, we need a way to use file_wrapper.chunks() to read
+            #         the file in chunks and stream that to Blockstore, but Blockstore
+            #         currently lacks an API for streaming file uploads.
+            #       Ref:  https://github.com/openedx/edx-platform/issues/34737
```

Comment:
We'll still need some kind of base checking to make sure it's not over our size limit, and we'll probably want to support streaming at some point.